openrarity/collection/collection.py

```diff
@@ -166,30 +241,49 @@ def rank_collection(
         ] = ("metric.unique_trait_count", "metric.information_entropy"),
         return_ranks: bool = True,
     ) -> list[RankedToken] | None:
-        """Preprocess tokens then rank the tokens for the collection and set the
-        corresponding cls.ranks attribute. Optionally, return the ranks.
-
+        """Preprocess tokens then rank the tokens for the collection and set the corresponding `cls.ranks` attribute. Optionally, return the ranks.
+
+        Notes
+        -----
+        metric.probability
+            The statistical probability/likelihood that a token exists in a collection.
+            Example : Flipping a coin for Heads
+                        -The likelihood that a coin turn to Heads
+        metric.information
+            The total information we gain from the token attributes.
+            Example : Flipping a coin for Heads
+                        -Knowledge you have to GAIN after the FLIP
+        metric.information_entropy
+            It is a measure of information in terms of uncertainty.
+            Example : Flipping a coin for Heads
+                        -Measure of uncertainty before FLIP
+        metric.unique_trait_count
+            It is the count of unique traits(specific properties each NFT has) in a token.
```

Comment:
It is traits that are unique across the entire collection though specifically. Such as a NFT having the only blue hat in the collection.